decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDecider.java

```diff
@@ -319,6 +338,10 @@ public void setConfig(Map<String, String> config) {
         return scheduledWorkflowRuns;
     }
 
+    public List<String> getErrors() {
+        return errors;
+    }
+
     public List<WorkflowRun> run() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 156 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)